pkg/rbac/common.go

```diff
@@ -42,7 +42,7 @@ const (
 	CrtbOwnerLabel                    = "authz.cluster.cattle.io/crtb-owner"
 	PrtbOwnerLabel                    = "authz.cluster.cattle.io/prtb-owner"
 	AggregationLabel                  = "management.cattle.io/aggregates"
-	clusterRoleOwnerAnnotation        = "authz.cluster.cattle.io/clusterrole-owner"
+	ClusterRoleOwnerLabel             = "authz.cluster.cattle.io/clusterrole-owner"
```

Comment:
Q: What happens on upgrade and rollback with this change from annotations to labels?